src/execution_plan/ops/shared/update_functions.c

```diff
@@ -272,6 +272,28 @@ void EvalEntityUpdates(GraphContext *gc, PendingUpdateCtx **node_updates,
 				array_append(*updates, update);
 			}
 			continue;
+		} else if(SI_TYPE(new_value) & (T_NODE | T_EDGE)) {
+			// value is a node or edge; perform attribute set reassignment
+			GraphEntity *ge = new_value.ptrval;
+			if(attr_id != ATTRIBUTE_ALL) {
```

Comment:
Why do we raise an error here but assert at line 259?

Reply:
Line 259 should also be an error, fixed now.